test/helpers/scripts/default-exceptions.js

```diff
@@ -8,10 +8,11 @@
 
 var path = require('path'),
     winston = require('../../../lib/winston');
+const testLogFixturesPath = path.join(__dirname, '..', '..', 'fixtures', 'logs');
```

Comment:
What is the benefit of this change? Why do you need to name a path to use it once?

Reply:
Consistency throughout all the test files.

Reply:
It was already consistent in all the files though. This change just stores that consistent value in a variable now. I don't see the benefit of a single-use variable in most cases like this.

Reply:
The `path.dir()` usage was not consistent, and in some cases never will be because it's building a path relative to the file location. Some files only need to navigate up a single parent directory, others have to navigate two. So within a given directory structure it should be consistent (post directory restructure). You are correct that in this specific file there's not a ton of gain, other than if someone needs to change the `path.join` usage they can easily search for the known constant that ever other file leverages/references. I personally view this as a magic string therefore putting it in a named const has value for the people who will eventually touch these files. Since this has no functional change and is purely affecting of the tests I'm inclined to leave as-is.

EG if we decide to enable parallel tests down the road (believe it's off by default with mocha) then we'll almost definitely run into issues with tests using the same directory for writing logs when using the file based transport. With this constant, it's now easier to determine which `path.join` usages are applicable to building that specific path at runtime; instead of having to parse every usage throughout the entire `./test/*` codebase.